src/pudl/metadata/fields.py

```diff
@@ -3060,6 +3070,11 @@
         "description": "Reactive Power Output (MVAr)",
         "unit": "MVAr",
     },
+    "reactive_power_rating": {
+        "type": "number",
+        "description": "The nameplate reactive power rating for the energy storage device.",
+        "unit": "MVAr",
+    },
```

Comment:
I think this is the same as the existing `reactive_power_output_mvar` column, but I'm not 100% sure.